packages/server/src/fhir/lookups/token.ts

```diff
@@ -113,13 +115,14 @@ export class TokenTable extends LookupTable<Token> {
       new Condition(new Column(table, 'id'), '=', new Column(joinName, 'resourceId')),
       new Condition(new Column(joinName, 'code'), '=', filter.code),
     ]);
-    joinOnExpression.expressions.push(buildWhereExpression(joinName, filter));
+    const whereExpression = buildWhereExpression(joinName, filter);
+    if (whereExpression) {
+      joinOnExpression.expressions.push(whereExpression);
+    }
+
     selectQuery.leftJoin(tableName, joinName, joinOnExpression);
 
-    // If the filter is "not equals", then we're looking for ID=null
```

Comment:
Please keep these comments, they are useful.

Reply:
Kept, just moved to line 212